### PR TITLE
Build gyp

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

File: packages/foreman/gyp/gyp.spec [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+%files
+%license LICENSE
+%doc AUTHORS
+%{_bindir}/gyp
+%{python3_sitelib}/*
+
+
+%changelog
+* Tue mar 24 2020 Zach Huntington-Meath <zhunting@redhat.com> - 0.1-0.33.fcd686f1git.2
+-
```
Comment (ekohl) [inline packages/foreman/gyp/gyp.spec]: What are the changes here?

Comment (zjhuntin) [reply packages/foreman/gyp/gyp.spec]: Thanks!

Comment (zjhuntin) [reply packages/foreman/gyp/gyp.spec]: I decided I wasn't changing it, so I just kept it as is for now. Should I go ahead and bump it just to show we are building it here?

**Comment by zjhuntin** [comment]
@ekohl yes, the patches are from the src rpm which came from https://centos.pkgs.org/8/epel-x86_64/gyp-0.1-0.33.fcd686f1git.el8.1.noarch.rpm.html

File: packages/foreman/gyp/gyp.spec (hunk 1 above)
Comment (ekohl) [reply packages/foreman/gyp/gyp.spec]: If there's no change at all, I'd keep the exact same version as in EPEL including the same release.

**Comment by ekohl** [comment]
I wonder if we should rebuild this or rather rely on EPEL since it saves us from updating it. Given there are patches, that's IMHO a strong reason not to copy it. Any idea how many more packages we'll need that are already in EPEL?

**Comment by zjhuntin** [comment]
I do not know exactly, @pcreech can you think of any off the top of your head?

**Comment by pcreech** [comment]
I don't have an immediate list off the top of my head.

We are being a little more intentional this time around with trying our best to not rely on epel if we can.

As far as management of the package, looks like this is a very stable codebase, both the source repository and in fedora's packaging repositories.  There hasn't been really any changes in months.

Personally, my vote is to continue down the no-epel path, at least a little bit further.

**Comment by zjhuntin** [comment]
@ehelms added

**Pull request merged**